site/layouts/index.redirects

```diff
@@ -0,0 +1,13 @@
+{{ $versions := site.Params.versions }}
+{{ $latest   := site.Params.latest }}
+/docs                          /docs/{{ $latest }}     301!
+/docs/latest                   /docs/{{ $latest }}
+/docs/latest/*                 /docs/{{ $latest }}/:splat
+/docs/troubleshooting          /docs/{{ $latest }}/troubleshooting
+/docs/supported-providers      /docs/{{ $latest }}/supported-providers
+/docs/zenhub                   /docs/{{ $latest }}/zenhub
+/docs/install-overview         /docs/{{ $latest }}/basic-install
+/docs/start-contributing       /docs/{{ $latest }}/start-contributing
+/docs/customize-installation   /docs/{{ $latest }}/customize-installation
+/docs/faq                      /docs/{{ $latest }}/faq
+/docs/csi                      /docs/{{ $latest }}/csi
```

Comment:
While we're at it, can we add `docs/restic`, since that's a big one?

Reply:
Added!